WORKSPACE

```diff
@@ -16,7 +16,7 @@ http_archive(
 load("@bazel_skylib//lib:versions.bzl", "versions")
 # Keep this version in sync with the BAZEL environment variable defined
 # in our .travis.yml config.
-versions.check(minimum_bazel_version = "0.22.0")
+versions.check(minimum_bazel_version = "0.26.1")
```

Comment:
Just to clarify, is 0.26.1 the minimum for any reasonable version of rules_nodejs?  Or if we went back a few versions on that, could we retain compatibility across more than one bazel version?

Reply:
> Just to clarify, is 0.26.1 the minimum for any reasonable version of rules_nodejs?

0.26.0 is the minimum version required by rules_nodejs.